### ISSUE TITLE
Space in username cause crash

### ISSUE THREAD

**Issue by quentinguidee** [Omega-Numworks/Omega#46]
#### Describe the bug
Put a space in username cause crash
```
gcc: error: G.: No such file or directory
make: *** [build/rules.mk:9: output/release/simulator/linux/ion/src/simulator/external/sdl/src/audio/dummy/SDL_dummyaudio.o] Error 1
make: *** Waiting for unfinished jobs....
gcc: error: G.: No such file or directory
make: *** [build/rules.mk:9: output/release/simulator/linux/ion/src/simulator/external/sdl/src/file/SDL_rwops.o] Error 1
gcc: error: G.: No such file or directory
make: *** [build/rules.mk:9: output/release/simulator/linux/ion/src/simulator/external/sdl/src/haptic/dummy/SDL_syshaptic.o] Error 1
```
#### Environment
 - Omega Version: 1.13.5

**Comment by quentinguidee** [comment]
Temporary workaround: `USERNAME="Your\ name\ here"`

**Comment by quentinguidee** [comment]
Closed with #47
